simulator/CARLA_Autoware/carla_pointcloud_interface/src/carla_pointcloud_interface/carla_pointcloud_interface_node.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Raditya.
+
+#include "carla_pointcloud_interface/carla_pointcloud_interface_node.hpp"
+
+#include <pcl_ros/transforms.hpp>
+
+#include <pcl_conversions/pcl_conversions.h>
+#include <yaml-cpp/yaml.h>
+
+#include <memory>
+
+void PointCloudInterface::processScan(const sensor_msgs::msg::PointCloud2::SharedPtr scanMsg)
+{
+  {
+    sensor_msgs::msg::PointCloud2 transformed_cloud;
+    if (pcl_ros::transformPointCloud(tf_output, *scanMsg, transformed_cloud, *tf_buffer_)) {
+      transformed_cloud.header.stamp = scanMsg->header.stamp;
+      velodyne_points_top->publish(transformed_cloud);
+      velodyne_points_con->publish(transformed_cloud);
+      velodyne_points_raw->publish(transformed_cloud);
```

Comment:
It is costly to publish the same data on 3 different topics.
Instead, it would be better to publish to one topic and remap the inputs of the subscribing nodes to that one topic. It is probably complicated to do so it can be fixed later.

Reply:
Fixed.
Removed `top` and `con` publishing but keeping the `raw`.
Added remapping through the `carla_autoware.launch.xml` file to remap `/points_raw` to other topic.